src/Workspaces/Core/Portable/CodeActions/CodeAction.cs

```diff
@@ -265,11 +274,8 @@ protected virtual Task<IEnumerable<CodeActionOperation>> ComputePreviewOperation
             return changedDocument.Project.Solution;
         }
 
-        internal virtual Task<Solution?> GetChangedSolutionAsync(
-            IProgressTracker progressTracker, CancellationToken cancellationToken)
-        {
-            return GetChangedSolutionAsync(cancellationToken);
-        }
+        internal virtual Task<Solution?> GetChangedSolutionAsync(IProgress<CodeAnalysisProgress> progress, CancellationToken cancellationToken)
+            => GetChangedSolutionAsync(cancellationToken);
```

Comment:
this is a method that would become purely protected once we agree that thsi api shape looks good.